project/plugins.sbt

```diff
@@ -1,13 +1,13 @@
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.20")
 
-addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.5.0")
-
 addSbtPlugin("com.typesafe.sbt" % "sbt-scalariform" % "1.3.0")
 
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "0.2.2")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8.3")
```

Comment:
this is obsolete / unsupported https://www.scala-sbt.org/sbt-pgp/index.html

Reply:
this is good to know, but not related with this PR, I'll keep it ATM